pkg/component/pushed_component.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
To reduce the number of changes in this PR, do we need this import alias?

From the coding conventions page ([`ImportAliasing`](https://github.com/redhat-developer/odo/wiki/Dev:-Coding-Conventions#import-aliasing)):

> Import aliasing must be used if the package name does not match the last element of the import path.
> In all other scenarios, import aliases should be avoided unless there is a direct conflict between imports.